lib/src/supabase_auth.dart

```diff
@@ -330,7 +330,8 @@ extension GoTrueClientSignInProvider on GoTrueClient {
     );
     final uri = Uri.parse(res.url!);
 
-    final willOpenWebview = authScreenLaunchMode == LaunchMode.inAppWebView &&
+    final willOpenWebview = (authScreenLaunchMode == LaunchMode.inAppWebView ||
+            authScreenLaunchMode == LaunchMode.platformDefault) &&
         context != null &&
         !kIsWeb && // `Platform.isIOS` throws on web, so adding a guard for web here.
```

Comment:
Allowing `platformDefault` as the value of authScreenLaucnMode as long as `context` is also passed.